Public/Sdk/Public/Deployment/filesystem.dsc

```diff
@@ -39,6 +57,152 @@ export interface DeployToDiskArguments {
     deploymentOptions?: DeploymentOptions;
 }
 
+@@public
+export function createDeployableOpaqueSubDirectory(opaque: OpaqueDirectory, sub: RelativePath): OpaqueSubDirectory {
+    return <OpaqueSubDirectory> {
+        opaque: opaque,
+        subDirectory: sub,
+        deploy: (
+            item: Object, 
+            targetFolder: RelativePath,
+            handleDuplicateFile: HandleDuplicateFileDeployment, 
+            result: FlattenedResult,
+            deploymentOptions?: Object,
+            provenance?: Diagnostics.Provenance) => 
+        {
+            const existingOpaque = result.flattenedOpaques.get(targetFolder);
+
+            if (existingOpaque !== undefined) {

```

Comment:
>existingOpaque [](start = 16, length = 14)

can this be undefined?

Reply:
This follows the same pattern as [here](https://github.com/microsoft/BuildXL/blob/42cf246cd46a20fce8e5b626732b73db8adb7c42/Public/Sdk/Public/Deployment/operations.dsc#L229)

That said, I'm not sure if this can be undefined.  I don't think it can.